feat(connectors): add google sheets connector docs (#2178)
camunda/camunda-platform-docs: docs/components/connectors/out-of-the-box-connectors/google-sheets.md

---
id: google-sheets
title: Google Sheets Connector
sidebar_label: Google Sheets Connector

> description: Use the Google Sheets Connector to connect your BPMN service with Google Sheets, and work with spreadsheets using this service.

Author: changed

---

> The **Google Sheets Connector** is an outbound Connector that allows you to create empty spreadsheet
> on [Google Drive](https://drive.google.com/) and work with it or already existing one from your BPMN process.

Reviewer: ```suggestion
The **Google Sheets Connector** is an outbound Connector that allows you to work with an existing or new empty spreadsheet
on [Google Drive](https://drive.google.com/) from your BPMN process.
```

Author: changed
